frappe/desk/doctype/event/event_list.js

```diff
@@ -1,5 +1,14 @@
 frappe.listview_settings["Event"] = {
-	add_fields: ["starts_on", "ends_on"],
+	add_fields: ["starts_on", "ends_on", "status"],
+	get_indicator: function(doc) {
+		if (doc.status === "Closed") {
+			return [__("Closed"), "green", "status,=,Closed"];
+		} else if (doc.status === "Completed") {
+			return [__("Completed"), "green", "status,=,Completed"];
+		} else if (doc.status === "Open") {
+			return [__("Open"), "red", "status,=,Open"];
+		}
+	},
```

Comment:
isn't this already present?
![Screenshot from 2022-10-26 13-48-18](https://user-images.githubusercontent.com/32034600/197972855-4e90b9a5-15b4-4956-8881-93d495d64756.png)

#

it also gets translated (if that's what  you were going for {?})
![Screenshot from 2022-10-26 13-52-08](https://user-images.githubusercontent.com/32034600/197973596-8124e1dc-b28d-47cf-b790-0bd64f782c88.png)